src/current/v24.2/install-cockroachdb-windows.md

```diff
@@ -24,76 +24,87 @@ Use one of the options below to install CockroachDB. To upgrade an existing clus
 
-Docker images for previous releases contain Intel binaries only. Intel binaries can run on ARM systems, but with a significant reduction in performance.
+Intel binaries can run on ARM systems, but with a significant reduction in performance.
 
 1. Install <a href="https://docs.docker.com/docker-for-windows/install/">Docker for Windows</a>.
 
     {{site.data.alerts.callout_success}}
-    Docker for Windows requires 64bit Windows 10 Pro and Microsoft Hyper-V. Please see the <a href="https://docs.docker.com/docker-for-windows/install/#what-to-know-before-you-install">official documentation</a> for more details. Note that if your system does not satisfy the stated requirements, you can try using <a href="https://docs.docker.com/toolbox/overview/">Docker Toolbox</a>.
+    Docker for Windows requires 64-bit Windows 10 Pro and Microsoft Hyper-V. Refer to the [official documentation](https://docs.docker.com/docker-for-windows/install/#what-to-know-before-you-install) for more details. If your system does not satisfy the stated requirements, you can try using [Docker Toolbox](https://docs.docker.com/toolbox/overview/).
     {{site.data.alerts.end}}
 
-1. In PowerShell, confirm that the Docker daemon is running in the background:
+1. In PowerShell, confirm that Docker is running in the background:
 
     {% include_cached copy-clipboard.html %}
     ~~~ powershell
     docker version
     ~~~
 
-    If you see an error, start Docker for Windows.
+    If you see an error, verify your Docker for Windows installation, then try again.
 
-1. <a href="https://docs.docker.com/docker-for-windows/#/shared-drives">Share your local drives</a>. This makes it possible to mount local directories as data volumes to persist node data after containers are stopped or deleted.
+1. [Enable synchronized drive sharing](https://docs.docker.com/desktop/synchronized-file-sharing/) so you can mount local directories as data volumes to persist node data after containers are stopped or deleted. so you can mount local directories as data volumes to persist node data after containers are stopped or deleted.
 
-1. In PowerShell, pull the image for the {{page.release_info.version}} release of CockroachDB from <a href="https://hub.docker.com/r/{{page.release_info.docker_image}}/" class="win-docker-step3" id="win-docker-step3-{{page.version.version}}" data-eventcategory="win-docker-step3">Docker Hub</a>:
+1. <a id="win-docker-step3-{{ page.version.name }}"></a>Pull the image for CockroachDB from [Docker Hub](https://hub.docker.com/layers/{{page.release_info.docker_image}}/). Releases are rolled out gradually, so the latest version may not yet be available. Using the `latest` tag is not recommended; to pull the latest release within a major version, use a tag like `latest-{{ page.version.version }}`. The following command always pulls the `{{ page.version.name }}` image.
```

Comment:
I still think 2 things could be clearer:

1. Why is the `latest` tag not recommended; is it related to Cloud-first potentially giving a 404?

1. The "latest version may not yet be available" point seems buried given that (I assume) it's meant to explain why someone might get a 404. If I'm understanding correctly then something like "latest version may not be available and may lead to a 404" would be clearest. If I'm not understanding then nevermind.

Reply:
1. `latest` tag is not recommended because it will replace your binary happily with the next major version without you intending to a download, and because it's not a recommended practice in the industry. `latest` means whatever a container producer means it to mean to them. Further, `latest` will get you v24.1 today instead of v24.2, which might be super confusing!
2. It may not yet be available during Cloud-first, and using `latest-v24.2` will not help if we are in the Cloud-first phase of v24.2.0, when there is no v24.2 Docker image available yet in any way. I can't solve that, so this sentence is an attempt to get them to go check on Docker Hub and grab the latest that they can, even if it's not yet for the version of the doc they are reading. The fully-qualified patch Docker tag will break during the Cloud-first phase of _every patch_, so I don't want to recommend it. Can you suggest an alternative?

Reply:
I see, that intent wasn't clear to me. This wording fixes that and echoes the way it's written under `Download the executable`. Wdyt?

```suggestion
1. <a id="win-docker-step3-{{ page.version.name }}"></a>Visit [Docker Hub](https://hub.docker.com/layers/{{page.release_info.docker_image}}/) and make a note of the full version of CockroachDB to pull. Releases are rolled out gradually, so the latest version may not yet be available. Using the `latest` tag is not recommended; to pull the latest release within a major version, use a tag like `latest-{{ page.version.version }}`. The following command always pulls the `{{ page.version.name }}` image.
```

Reply:
Done in all versions